js/text_encoding_test.ts

```diff
@@ -65,17 +54,6 @@ test(function textDecoderErrorEncoding() {
   assert(didThrow);
 });
 
-test(function textEncoder() {
-  const fixture = "������";
```

Comment:
This is likely due to ea6c9f2f365698e8120177bb7a1344e83f859180 .. i regret landing it now because I think it was actually incorrect. will attempt to revert the revert and see if this is fixed.

Reply:
Hmmmm... this particular string is specifically designed to test the upper limit of high-surrogate pairs.  If there was some sort of encoding issue, injecting mis-encoded stuff from Rust could cause this issue.  I double checked just now in Chrome and the test is correct in that the encoding of the bytes of the string is right.

Reply:
Ya it’s working as intended and breaking something :) not sure what it is tho ...

Reply:
I would suggest to keep these tests (unless there's actually something wrong with them).